code/modules/antagonists/abductor/equipment/glands/viral.dm

```diff
@@ -24,8 +24,8 @@
 		EFFECT_DANGER_HARMFUL	= 0,
 		EFFECT_DANGER_DEADLY	= 0,
 	)
-	var/virus_choice = pick(subtypesof(/datum/disease/advanced)- typesof(/datum/disease/advanced/premade))
-	var/datum/disease/advanced/D = new virus_choice
+	var/virus_choice = pick(subtypesof(/datum/disease/acute)- typesof(/datum/disease/acute/premade))
```

Comment:
honestly with how much this pattern is repeated, I'd prefer a define somewhere like this:
```
#define NON_PREMADE_ACUTE_DISEASES (subtypesof(/datum/disease/acute) - typesof(/datum/disease/acute/premade))
```

altho you can prolly come up with a better name for it